package mpsqual;

message HostDomainDeviceServiceRequest {
    enum RequestTypeT {
        GET = 0;
        SET = 1;
    }

    message PropertyReq {
        required string key = 1;
        optional string value = 2;
    }

    required RequestTypeT requestType = 1;
    repeated PropertyReq values = 2;
}

message HostDomainDeviceServiceResponse {
    message PropertyResp {
        required bool success = 1;
        required string key = 2;
        required string value = 3;
    }

    repeated PropertyResp values = 1;
}